### PR TITLE
Import of legacy transfers

### PR DESCRIPTION
**Pull request by catalinjitea**
refs #925

### CODE REVIEW

File: ozone/core/management/commands/import_transfers.py [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+            for entry in entries:
+                success_count += self.delete_instance(
+                    party=Party.objects.get(id=entry['submission']['party_id']),
+                    period=ReportingPeriod.objects.get(id=entry['submission']['reporting_period_id']),
+                    obligation=Obligation.objects.get(id=4)
+                )
+            return success_count
+
+        for entry in entries:
+            import pdb; pdb.set_trace()
```
Comment (melish) [inline ozone/core/management/commands/import_transfers.py]: pdb

**Comment by melish** [comment]
There seem to be some duplicates in the import log, e.g.:
2019-05-09 19:09:12,866 INFO Submission FR/1994 imported.
2019-05-09 19:09:12,899 INFO Submission FR/1994 imported.
...
2019-05-09 19:09:15,377 INFO Success on 86 out of 86

While the source file contains only 80 rows in ProdTransfers sheet.

Not 100% sure, but there should be 51 total submissions. Now there are 86 submissions and 125 rows in reporting_transfer.

Extra check: there should be 2 submissions having transferred_amount=1500 (now there are 4).

**Comment by sorinstelian** [comment]
Aggregations will be added later as part of the solution for https://github.com/eaudeweb/ozone/issues/948

**Comment by sorinstelian** [comment]
@catalinjitea : Due to the solution for #952, you can now call `fill_aggregated_data()` for this type of submissions.

**Comment by sorinstelian** [comment]
@melish : please re-review :)

Also, please note that this requires running:
`python manage.py loaddata obligations.json`

**Pull request merged**